SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -246,6 +246,7 @@ class USpatialReceiver : public UObject, public SpatialOSDispatcherInterface
 	TMap<Worker_EntityId_Key, TWeakObjectPtr<USpatialNetConnection>> AuthorityPlayerControllerConnectionMap;
 
 	TMap<TPair<Worker_EntityId_Key, Worker_ComponentId>, PendingAddComponentWrapper> PendingDynamicSubobjectComponents;
+	TMap<Worker_EntityId_Key, FString> WorkerConnectionEntity;
```

Comment:
```suggestion
	TMap<Worker_EntityId_Key, FString> WorkerConnectionEntities;
```
I believe this should be plural to indicate it's a map.